tools/show_ox_upstairs.sh

```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
 
-UUID=4f0d2ac9-dde5-44cf-9011-12add9cd975a
-# Show me the stats for a hard coded upstairs UUID
+if [[ -n $1 ]]; then
+    UUID=$1
+else
+    # Use this hard coded UUID, which is probably not what you want.
+    UUID=2effac35-a8ba-45ae-966d-4487e0df0ad6
```

Comment:
`d7a01b55-93a6-c873-b341-d03c93385e97`

Reply:
Removing the default UUID, so we don't have to argue.